Jenkinsfile.release

```diff
@@ -85,9 +85,21 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
             cosa init --branch ${params.STREAM} https://github.com/coreos/fedora-coreos-config
             cosa buildfetch --build=${params.VERSION} \
                 --arch=all --url=s3://${s3_stream_dir}/builds
+            cosa buildprep --ostree --build=${params.VERSION} \
+                --arch=all s3://${s3_stream_dir}/builds
             """)
         }
 
+        stage('Sync oscontainer to quay.io') {
+            withCredentials([string(credentialsId: "oscontainer-secret", variable: "OSCONTAINER_SECRET")]) {
+                shwrap("""ociarchive=\$(cosa meta --image-path ostree)
+                        case \${ociarchive} in
+                        *ociarchive) skopeo copy --authfile=<($OSCONTAINER_SECRET) oci-archive://\${ociarchive} docker://quay.io/coreos-assembler/fcos:${params.STREAM};;
```

Comment:
Might be good to const-ify the quay.io URL here at the beginning of the file for clarity. And also a comment there that says long-term we want `quay.io/fedora/coreos` as per https://fedoraproject.org/wiki/Changes/OstreeNativeContainer.